### PR TITLE
Add settings to disable unwanted indicators.

### PR DESCRIPTION
**Pull request by roidy**
Add settings to disable unwanted indicators, in order to make the
display cleaner.

### CODE REVIEW

File: resources/lib/vfdsettings.py [hunk 1]
```diff
@@ -116,6 +143,15 @@ def readValues(self):
...
 			self._displayBrightness = getSettingInt('display.brightness')
+			self._hdmiIndicator = getSettingBool('display.hdmi.indicator')
+			self._cvbsIndicator = getSettingBool('display.cvbs.indicator')
+			self._ethIndicator = getSettingBool('display.eth.indicator')
+			self._wifiIndicator = getSettingBool('display.wifi.indicator')
+			self._setupIndicator = getSettingBool('display.setup.indicator')
+			self._appsIndicator = getSettingBool('display.apps.indicator')
+			self._usbIndicator = getSettingBool('display.usb.indicator')
+			self._sdIndicator = getSettingBool('display.sd.indicator')
+			self._powerIndicator = getSettingBool('display..indicator')
```
Comment (arthur-liberman) [inline resources/lib/vfdsettings.py]: I think you missed 'power' here.

File: resources/settings.xml [hunk 2]
```diff
@@ -3,9 +3,21 @@
 	<category label="30000">
 		<setting label="30001" type="bool" id="display.on" default="true" />
 		<setting label="30002" type="slider" id="display.brightness" default="7" range="0,7" option="int" enable="eq(-1,true)" />
-		<setting label="30003" type="bool" id="display.storage.indicator" default="false" enable="eq(-2,true)" />
-		<setting label="30004" type="labelenum" id="display.storage.indicator.icon" lvalues="30100|30101|30102|30103|30104|30105|30106|30107|30108|30109|30110|30111" default="10" enable="eq(-3,true) + eq(-1,true)" />
-		<setting label="30005" type="bool" id="display.colon.on" default="false" enable="eq(-4,true)" />
+		<setting label="30003" type="bool" id="display.colon.on" default="false" enable="eq(-6,true)" />
```
Comment (arthur-liberman) [inline resources/settings.xml]: Should be eq(-2,true)

File: resources/settings.xml [hunk 3]
```diff
@@ -3,9 +3,21 @@
...
+		<setting label="30102" type="bool" id="display.cvbs.indicator" default="true" enable="eq(-2,true)" />
+		<setting label="30103" type="bool" id="display.eth.indicator" default="true" enable="eq(-3,true)" />
+		<setting label="30104" type="bool" id="display.wifi.indicator" default="true" enable="eq(-4,true)" />
+		<setting label="30105" type="bool" id="display.setup.indicator" default="true" enable="eq(-5,true)" />
+		<setting label="30106" type="bool" id="display.apps.indicator" default="true" enable="eq(-6,true)" />
+		<setting label="30107" type="bool" id="display.usb.indicator" default="true" enable="eq(-7,true)" />
+		<setting label="30108" type="bool" id="display.sd.indicator" default="true" enable="eq(-8,true)" />
+		<setting label="30109" type="bool" id="display.power.indicator" default="true" enable="eq(-9,true)" />
+		<setting label="30110" type="bool" id="display.storage.indicator" default="false" enable="eq(-10,true)" />
+		<setting label="30111" type="labelenum" id="display.storage.indicator.icon" lvalues="30200|30201|30202|30203|30204|30205|30206|30207|30208|30209|30210|30211" default="10" enable="eq(-12,true) + eq(-1,true)" />
```
Comment (arthur-liberman) [inline resources/settings.xml]: eq(-11,true), not -12.

**Comment by arthur-liberman** [comment]
Thanks for the PR. I haven't considered making the indicators optional. Please note my comments.

**Comment by roidy** [comment]
Good catch, thanks.

**Review by arthur-liberman — APPROVED** [review]

**Pull request merged**